assets/src/editor/blocks/card/index.js

```diff
@@ -12,13 +12,15 @@ import { __ } from '@wordpress/i18n';
 import BlockIcon from '../../../svg/blocks/card.svg';
 import CallToActionPicker from '../../components/cta';
 import ImagePicker from '../../components/image-picker';
+import sharedStyles from '../../helpers/block-styles';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-export const name = 'shiro/card';
+export const name = 'shiro/card',
+	styles = sharedStyles;
```

Comment:
Nit: Slight stylistic preference for making these two separate statements, rather than using a comma. Comma-separated values seem to be less common with let and const (and export) than they used to be with var.